projects/laji/src/app/shared-modules/spreadsheet/importer/importer.component.ts

```diff
@@ -436,7 +438,10 @@ export class ImporterComponent implements OnInit, OnDestroy {
       publicityRestrictions
     }).pipe(
       switchMap(() => this.importService.waitToComplete('create', this.jobPayload, (status) => {
-        this.current = status.processed;
+        ticker += add;
+        this.current = status.processed === this.total ?
+          status.processed :
+          Math.min(this.total - 1, ticker);
```

Comment:
`this.current` might end up being -1, is that intentional?

If `this.total === 0`, the latter operation of the ternary operation is executed, and `Math.min` between values `0 - 1` (this.total - 1`) and `>0` (`ticker`) will be `-1`.

Reply:
`this.total` is the number of rows in the excel and there should always be some lines in it. `this.current` is only used on progress bars